contents/blog/best-survicate-alternatives.mdx

```diff
@@ -0,0 +1,436 @@
+---
+title: 'The best Survicate alternatives & competitors, compared'
+date: 2024-06-18
+author:
+  - lior-neu-ner
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-alternatives/posthog-alternatives.jpeg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+## 1. PostHog
+
+- **Founded:** 2020
+- **Similar to:** Hotjar, Pendo
+- **Typical users:** Engineers and product teams
+
+![PostHog](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/screenshots/feature-flags.png)
+
+### What is PostHog?
+
+[PostHog](/) (that's us 👋) is an open-source platform that combines surveys, product analytics, session replay, A/B testing, and feature flags. This means it's not only an alternative to Survicate, but also tools like [Mixpanel](/blog/posthog-vs-mixpanel) and [LaunchDarkly](/blog/posthog-vs-launchdarkly).
+
+According to [BuiltWith](https://trends.builtwith.com/analytics/PostHog), as of June 2024, 4,661 of the top million websites deploy PostHog, which is more than the 978 that use Survicate.
+
+#### Key features
+
+- 💬 **Surveys:** Target surveys by event or user properties. Templates for [net promoter score (NPS)](/templates/nps-survey), [product-market fit (PMF)](/templates/pmf-survey) surveys, and more.
+
+- 📈 **Product analytics:** Custom trends, funnels, user paths, retention analysis, and segment user cohorts. Also, direct [SQL querying](/docs/product-analytics/sql) for power users.
+
+- 📺 **Session replays:** View exactly how users are using your site. Includes event timelines, console logs, network activity, and 90-day data retention.
+
+- 🧪 **A/B tests:** Experiment in your app with up to nine test variations and track impact on primary and secondary metrics. Auto-calculate test duration, sample size, and statistical significance.
+
+- 🚩 **Feature flags:** Rollout features safely with [local evaluation](/docs/feature-flags/local-evaluation) (for faster performance), JSON payloads, and instant rollbacks.
+
+### How does PostHog compare to Survicate?
+
+PostHog and Survicate have similar capabilities when it comes to surveys but differ on other features. PostHog includes entire features Survicate doesn't have, like product analytics, A/B testing, and session replays. 
+
+On the other hand, Survicate supports sending surveys on more platforms, such as mobile app, emails, and shareable-links. PostHog doesn't yet offer surveys on mobile – although all of its other features are available on apps.
+
+PostHog and Survicate both offer integrations so that you can send your survey data to your other tools. However, PostHog's integrations requires basic technical setup using [webhooks](/docs/webhooks), whereas Survicates integrations are all 1-click.
```

Comment:
Would be good to tighten this up a little. Also, feels contradictory to say we have similar survey features then follow that up with a bunch of survey features that are unique to Survicate.